packages/adminrouter/extra/src/README.md

```diff
@@ -24,9 +24,24 @@ Admin Router runs on both master and agent nodes, each with different configurat
 - HMTL: [docs/api/nginx.agent.html](docs/api/nginx.agent.html)
 - Rendered: <https://rawgit.com/dcos/dcos/master/packages/adminrouter/extra/src/docs/api/nginx.agent.html>
 
-Use `make api-docs` to regenerate the YAML and HTML files.
+## Endpoints documentation
 
-Use `make check-api-docs` to validate that the YAML and HTML files are up to date.
+All Admin Router endpoints are documented using [nginxdox](https://github.com/karlkfi/ngindox)
+tool, which uses special format of the comments in order to describe endpoint
+configuration, and automatically parse it into HTML documents. Please check the
+project documentation for more details.
+
+Admin Router CI automatically checks if the endpoint documentation generated
+using nginxdox and embedded into the repository is up to date. If not, the CI
+job is failed and the user needs to regenerate the docs and re-submit the PR.
+
+The check is done by generating the documentation also during the build stage.
+If, after the nginxdox run, git detects uncommitted changes, then this means
```

Comment:
I'm not sure if there is a standard but if you agree with the general rule "computer-y terms which should not be in the dictionary but are valid should be surrounded by backticks" then please surround "nginxdox" and "git". Else, I'd like to understand when backticks are used.